src/engine/controls/cuecontrol.cpp

```diff
@@ -883,6 +891,54 @@ void CueControl::hotcueSet(HotcueControl* pControl, double value, HotcueSetMode
         }
     }
 
+    // EveCue-Loop
+    bool TrackStem = false;
+    if (ControlObject::exists(ConfigKey(getGroup(), "stem_count"))) {
+        PollingControlProxy proxyStem(getGroup(), "stem_count");
```

Comment:
we should avoid CO look up during Mixxx runtime, because this is a looking operation. Can you move this into the constructor or somewhere else? Checking for exist already creates the CO half way. 
You can instead create the PollingProxy with ControlFlag::AllowMissingOrInvalid flags.

Reply:
fixed